### PR TITLE
Also allowing callables in Response::body() to return a string(ish)

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: src/Network/Response.php [hunk 1]
```diff
@@ -546,10 +546,10 @@ protected function _sendHeader($name, $value = null)
     protected function _sendContent($content)
     {
         if (!is_string($content) && is_callable($content)) {
-            call_user_func($content);
-        } else {
-            echo $content;
+            $content = $content();
         }
+
+        echo $content;
```
Comment (markstory) [inline src/Network/Response.php]: This could result in additional `1` being output if the callable returns `true`.

Comment (ADmad) [inline src/Network/Response.php]: Well the users should be careful not to do so :)

Comment (markstory) [inline src/Network/Response.php]: Ok.

Comment (ADmad) [inline src/Network/Response.php]: @markstory I made a PR #6964 clearing that in docblock.

Comment (markstory) [inline src/Network/Response.php]: :metal:

**Pull request merged**